clickable/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jazz2",
+    "name": "jazz2.dubix",
```

Comment:
I just guessed that. What are you actually using? The second part of the name identifies you in the OpenStore.

Reply:
But this will only allow access to ~/.local/share/jazz2.dubix and ~/.config/jazz2.dubix, but not to ~/.local/share/jazz2 and ~/.config/jazz2 right?

Reply:
That is right. But that is also where the app will write/read files if you configure it to via the `NCINE_LINUX_PACKAGE` parameter.